### PR TITLE
tg.js - Update 5.7.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by callmedandann** [comment]
Line 567: Bumalik sa huling checkpoint
Line 569: Lumipad
Line 571: Patayin
Line 685: Mga papremyo 
Line 699: Walang nahanap na laro<br><small>I-try na ibahin ang Quickplay options</small>

**Comment by callmedandann** [comment]
Line 567: Bumalik sa huling checkpoint
Line 569: Lumipad
Line 571: Patayin
Line 685: Mga papremyo
Line 699: Walang nahanap na laro
I-try na ibahin ang Quickplay options

**Comment by Jaehnvnct** [comment]
Line 754: Itago ang mga username (Allies)
Line 883: Hindi makapag-host ng laro habang nasa Tayaan
Line 1009: Malapit lang ang gintong itlog
Line 1010: Nawala na ang gintong itlog
Line 1012: Nakita mona ang itlog na ito!
Line 1013: Natapos mo ang paghahanap ng itlog!

**Comment by Navsy17** [comment]
Navsy  - 5
Dandan - 5
Jaehn - 1 (+4 for still contributing Golden egg changes)
Kebbs - 24 

total (July 11): 39 lines have been changed

**Comment by Markzt** [comment]
line 57 : ilagay ang password
line 684 : upang makumpleto ang hamon

**Comment by Markzt** [comment]
lines 683 : mga hamon

**Comment by Navsy17** [comment]
still ongoing, been busy with other stuff sorry for slow updates

**Comment by Navsy17** [comment]
Navsy - 58
Dandan - 5
Jaehn - 1 (+4 for still contributing Golden egg changes)
Kebbs - 24
Markzt - 3

File: tg.js [hunk 1]
```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
 purchase.error.click dito</a> subukan muli!
 purchase.error.limit Maaaring lumagpas ka sa iyong pang-araw-araw na limitasyon sa transaksyon. 
 settings.reset Mag-reset
-settings.reset.confirm Sigurado ka bang mag-reset ng inyong settings?
+settings.reset.confirm Sigurado ka ba na i-reset ang inyong settings?
 settings.import iangkat
```
Comment (StarRayX) [inline tg.js:81]:
```suggestion
settings.import i-import
```

File: tg.js [hunk 2]
```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
...
 settings.export iluwas
```
Comment (StarRayX) [inline tg.js:82]:
```suggestion
settings.export iexport
```

Not used in proper context, pag-angkat and pag-luwas is for trade/economic contexts

File: tg.js [hunk 3]
```diff
@@ -279,7 +279,7 @@ settings.crosshairm.color Kulay
 settings.crosshairm.shadow.color Shadow Color
 settings.crosshairm.shadow.thickness Shadow Thickness
 settings.crosshairm.size Laki
-settings.crosshairm.thickness Thickness
+settings.crosshairm.thickness Gaano Kakapal
```
Comment (StarRayX) [inline tg.js:282]:
```suggestion
settings.crosshairm.thickness Kakapalan
```

File: tg.js [hunk 4]
```diff
@@ -830,8 +830,8 @@ social.market.gift Magregalo ng KR
 social.market.gift.msg Mag-iwan ng mensahe
 respawn.auto Auto Respawning sa {0}
```
Comment (StarRayX) [inline tg.js:831]:
```suggestion
respawn.auto Kusang Magrerespawn sa {0}
```

File: tg.js [hunk 5]
```diff
@@ -830,8 +830,8 @@ social.market.gift Magregalo ng KR
...
 custom.host.ranked Hindi ma-host habang naglalaro sa 'Ranked'
-custom.host.wager Can't host match while in Wager.
-funds.insufficient Hindi sapat na KR
+custom.host.wager Hindi pwede mag-host habang mayroong wager
+funds.insufficient Hindi sapat ang KR
 funds.out Ubos ang KR
```
Comment (StarRayX) [inline tg.js:835]:
```suggestion
funds.out Ubos na ang iyong KR.
```

"It appears you have run out of Krunkies." was the original, it included second-person perspective

**Review by StarRayX — APPROVED** [review]

File: tg.js [hunk 6]
```diff
@@ -130,7 +130,7 @@ settings.chat.chatBGOp Chat BG Opacity
 settings.chat.chatHeight Tangkad ng Chat
 settings.chat.chatTimer Orasan ng Pag-chat
```
Comment (StarRayX) [inline tg.js:131]:
```suggestion
settings.chat.chatTimer Pagitan ng Pag-chat
```

File: tg.js [hunk 7]
```diff
@@ -130,7 +130,7 @@ settings.chat.chatBGOp Chat BG Opacity
...
 settings.chat.chatTextOutline Chat Text Outline
```
Comment (StarRayX) [inline tg.js:132]:
```suggestion
settings.chat.chatTextOutline Outline ng Chat Text
```

File: tg.js [hunk 8]
```diff
@@ -77,10 +77,10 @@ purchase.error May naganap na error
 purchase.error.click dito</a> subukan muli!
 purchase.error.limit Maaaring lumagpas ka sa iyong pang-araw-araw na limitasyon sa transaksyon. 
 settings.reset Mag-reset
-settings.reset.confirm Sigurado ka bang mag-reset ng inyong settings?
+settings.reset.confirm Sigurado ka ba na i-reset ang inyong settings?
 settings.import iangkat
 settings.export iluwas
-settings.none Walang Nahanap na Mga settings
+settings.none Hindi matagpuan ang settings
```
Comment (StarRayX) [inline tg.js:83]:
```suggestion
settings.none Hindi mahanap ang settings
```

File: tg.js [hunk 9]
```diff
@@ -279,7 +279,7 @@ settings.crosshairm.color Kulay
 settings.crosshairm.shadow.color Shadow Color
 settings.crosshairm.shadow.thickness Shadow Thickness
 settings.crosshairm.size Laki
-settings.crosshairm.thickness Thickness
+settings.crosshairm.thickness Gaano Kakapal
 settings.crosshairm.gap Gap
```
Comment (StarRayX) [inline tg.js:283]:
```suggestion
settings.crosshairm.gap Pagitan
```

File: tg.js [hunk 10]
```diff
@@ -308,12 +308,12 @@ settings.gameprivacy.header Pagkapribado ng Laro
 settings.gameprivacy.streamer Mode ng Streamer
```
Comment (StarRayX) [inline tg.js:308]:
```suggestion
settings.gameprivacy.streamer Streamer Mode
```
for uniformity with anonymous mode

File: tg.js [hunk 11]
```diff
@@ -308,12 +308,12 @@ settings.gameprivacy.header Pagkapribado ng Laro
...
 settings.gameprivacy.anon Anonymous Mode
 settings.gameprivacy.hideGameTitle Itago ang Pamagat ng Laro
-settings.gameprivacy.showRegion Show Region Info
+settings.gameprivacy.showRegion Ipakita ang Detalye ng Rehiyon
 settings.gameprivacy.verified Ipakita: Verification Badge
 settings.gameprivacy.premium Ipakita: Premium Badge
 settings.gameprivacy.partner Ipakita: Partnered Badge
 settings.gameprivacy.rank Ipakita: Ranked Badge
-settings.gameprivacy.custom Ipakita: Custom Badge
+settings.gameprivacy.custom Ipakita: Kustom Badge
```
Comment (StarRayX) [inline tg.js:316]:
```suggestion
settings.gameprivacy.custom Ipakita: Custom Badge
```
please revert, there's no need to change the c in custom on this line when there are 100 other lines with 'Custom' and not 'Kustom'

File: tg.js [hunk 12]
```diff
@@ -349,8 +349,8 @@ settings.audio.sound Bolyum o Tunog Pangkalahatan
 settings.audio.ambient Bolyum ng Ambiyansa
 settings.audio.voice Bolyum ng Boses
 settings.audio.input Audio/Mic Input
-settings.audio.mic Mic Volume
-settings.audio.micquality Mic Quality
+settings.audio.mic Bolyum ng Mic
+settings.audio.micquality Kalidad ng Mic
 settings.audio.weps Bolyum ng Tunog ng Armas
```
Comment (StarRayX) [inline tg.js:354]:
```suggestion
settings.audio.weps Bolyum ng mga Armas
```

File: tg.js [hunk 13]
```diff
@@ -680,9 +680,9 @@ windows.client.linux Installer sa Linux
 windows.ranked.header Rango
 windows.ranked.login Login upang magpataasan ng inyong rango!
 windows.ranked.level Hindi ka bababa sa Antas na <span color = "black"> level {0} </span> upang maglaro sa may ranggo. Ikaw ay kasalukuyang nasa Antas <span color = "black"> level {1} ​​</span>.
-windows.chall.header Mga Challenge
-windows.chall.login upang makumpleto ang mga challenge!
-windows.spin.header Prize Wheel
+windows.chall.header Mga Hamon
+windows.chall.login maglog-in upang makumpleto ang mga hamon!
+windows.spin.header Mga Papremyo
 thumbnail.upload.limit Failed. {0}kb Thumbnail Limit
```
Comment (StarRayX) [inline tg.js:686]:
```suggestion
thumbnail.upload.limit Failed. {0}kb Limitasyon sa mga Thumbnail```

File: tg.js [hunk 14]
```diff
@@ -680,9 +680,9 @@ windows.client.linux Installer sa Linux
...
 image.upload.limit Failed. {0}kb ang Limitasyon ng Larawan
```
Comment (StarRayX) [inline tg.js:687]:
```suggestion
image.upload.limit Failed. {0}kb ang Limitasyon sa mga Larawan
```

File: tg.js [hunk 15]
```diff
@@ -738,7 +738,7 @@ server.config.t3Dmg Damage ng Pangkat 3
 server.config.t4Dmg Damage ng Pangkat 4
 server.config.t5Dmg Damage ng Pangkat 5
 server.config.selTeam Piliin Ang Pangkat
```
Comment (StarRayX) [inline tg.js:740]:
```suggestion
server.config.selTeam Pumili ng Pangkat 
```

File: tg.js [hunk 16]
```diff
@@ -738,7 +738,7 @@ server.config.t3Dmg Damage ng Pangkat 3
 server.config.t4Dmg Damage ng Pangkat 4
 server.config.t5Dmg Damage ng Pangkat 5
 server.config.selTeam Piliin Ang Pangkat 
-server.config.frFire Friendly Fire
+server.config.frFire Pwedeng Barilin ang Kakampi
 server.config.allowSpect Panonood ng Laro
 server.config.killRewards Gantimpala kapag nakapatay
 server.config.headshotOnly Mga headshot Lamang
```
Comment (StarRayX) [inline tg.js:744]:
```suggestion
server.config.headshotOnly Mga Headshot Lamang
```

File: tg.js [hunk 17]
```diff
@@ -738,7 +738,7 @@ server.config.t3Dmg Damage ng Pangkat 3
 server.config.t4Dmg Damage ng Pangkat 4
 server.config.t5Dmg Damage ng Pangkat 5
 server.config.selTeam Piliin Ang Pangkat 
-server.config.frFire Friendly Fire
+server.config.frFire Pwedeng Barilin ang Kakampi
 server.config.allowSpect Panonood ng Laro
 server.config.killRewards Gantimpala kapag nakapatay
```
Comment (StarRayX) [inline tg.js:743]:
```suggestion
server.config.killRewards Gantimpala kapag may napatay
```

File: tg.js [hunk 18]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
 commands.suspect.menu Ang suspek ay nasa menu
 commands.suspect.left Ang suspek ay umalis
 commands.suspect.found Nahanap na ang suspek. nanonood: {0}, spawned: {1}, pangalan: {2}
-commands.forfeit.next Kailangan maglaro muna ng 1 laban
+commands.forfeit.next Kailangan maglaro muna ng isang beses
 commands.forfeit.wait Hintayin munang magsimula ang laro upang maka-forfeit 
 commands.forfeit.later Ulitin nalang muli mamaya
 commands.forfeit.voted Ikaw ay bumoto na.
```
Comment (StarRayX) [inline tg.js:999]:
```suggestion
commands.forfeit.voted Nakaboto ka na.
```

File: tg.js [hunk 19]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
-commands.forfeit.limit Isang beses kalang makababoto sa isang laro
+commands.forfeit.limit Isang beses kalang makaboboto sa isang laro
 commands.forfeit.ended Iniwan na ang laro
```
Comment (StarRayX) [inline tg.js:1001]:
```suggestion
commands.forfeit.ended Sumuko na sa laro
```

File: tg.js [hunk 20]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 commands.forfeit.ended.other Umalis na ang kabilang grupo
```
Comment (StarRayX) [inline tg.js:1002]:
```suggestion
commands.forfeit.ended.other Sumuko na ang kalaban
```

File: tg.js [hunk 21]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 server.message.join si {0} ay sumali sa laro
```
Comment (StarRayX) [inline tg.js:1003]:
```suggestion
server.message.join Si {0} ay sumali sa laro
```

File: tg.js [hunk 22]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 server.message.left si {0} ay umalis sa laro
```
Comment (StarRayX) [inline tg.js:1004]:
```suggestion
server.message.left Si {0} ay umalis sa laro
```

File: tg.js [hunk 23]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 server.message.kicked si {0} ay natanggal sa laro
```
Comment (StarRayX) [inline tg.js:1005]:
```suggestion
server.message.kicked Si {0} ay natanggal sa laro
```

File: tg.js [hunk 24]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 server.message.banned si {0} ay na-ban
```
Comment (StarRayX) [inline tg.js:1006]:
```suggestion
server.message.banned Si {0} ay na-ban
```

File: tg.js [hunk 25]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 server.message.votekicked na vote-kick si {0}
```
Comment (StarRayX) [inline tg.js:1007]:
```suggestion
server.message.votekicked Na-vote kick si {0} 
```

File: tg.js [hunk 26]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
 server.message.kicked si {0} ay natanggal sa laro
 server.message.banned si {0} ay na-ban
 server.message.votekicked na vote-kick si {0} 
-server.message.arrested si {0} ay nahuli ng K.P.D
-event.easter.golden.spawn A Golden egg is near
-event.easter.golden.despawn A Golden egg has gone
-event.easter.found.new {0} found {1}/{2} Eggs
-event.easter.found.old You already discovered this egg!
-event.easter.completed {0} completed the Egg hunt!
+server.message.arrested si {0} ay naaresto ng K.P.D
```
Comment (StarRayX) [inline tg.js:1008]:
```suggestion
server.message.arrested Si {0} ay naaresto ng K.P.D
```

File: tg.js [hunk 27]
```diff
@@ -993,21 +993,21 @@ commands.suspect.invalid walang valid na inpormasyon ang suspek
...
+event.easter.golden.spawn Nagkaroon ng Gintong Itlog
+event.easter.golden.despawn Nawala ang Gintong Itlog
+event.easter.found.new Si {0} aynakahanap ng {1}/{2} na Gintong Itlog
```
Comment (StarRayX) [inline tg.js:1011]:
```suggestion
event.easter.found.new Nakahanap si {0} ng {1}/{2} na Gintong Itlog
```

**Review by StarRayX — CHANGES_REQUESTED** [review]

File: tg.js (hunk 6 above)
Comment (Navsy17) [reply tg.js:131]: actually a good alternative than going literal, thanks alot

**Comment by Navsy17** [comment]
Navsy - 57
Dandan - 5
Jaehn - 1 (+4 for still contributing Golden egg changes)
Kebbs - 24
Markzt - 3
Goos/StarRayX - 27

**Comment by Navsy17** [comment]
Navsy - 70
Dandan - 5
Jaehn - 4
Kebbs - 24
Markzt - 3
Goos/StarRayX - 27

**Comment by mty22** [comment]
@kebbs please reach out to me with your ign.

**Comment by mty22** [comment]
your total came to 102% lol. it's fine though.

**Pull request merged**